src/components/Help.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+
+const Help = (): ReactElement => {
+    return (
+        <>
+            <legend className="govuk-fieldset__legend govuk-fieldset__legend--s">
+                <h1 className="govuk-fieldset__heading">Help and Support</h1>
+            </legend>
+            <p className="govuk-body">
+                If you are having problems, please contact the Fares Data Build service through the link below:
+                <div>
+                    <a href="/contact" className="govuk-link">
+                        Contact the FDBT team
```

Comment:
Also, should this now be FDB (Fares Data Build) team? Or maybe FDBS (Fares Data Build Service) team?

Reply:
Have discussed and amended in ticket